### ISSUE TITLE
PM2 doesn't send node v8 -harmony flag.

### ISSUE THREAD

**Issue by rogr** [Unitech/pm2#1808]
I can't run my application with pm2 and --harmony flag.
My configuration file:
```
{ "apps" : [{
    "name"        : "my-api",
    "script"      : "server/server.js",
    "node_args"   : "--harmony",
    "env": {
        "NODE_ENV": "production"
    }
  }]
}
```
Then I execute: `pm2 start pm2.json` and then i got `SyntaxError: Unexpected identifier` this is  because I use `let`.

The same if I execute: `pm2 start server/server.js --name my-api --node-args="--harmony"`

If i use `--next-gen-js` I get the next error: `error: unknown option '--next-gen-js'`

But if I run: `node --harmony server/server.js` everything is fine.

node --version: v0.12.7
pm2 --version: 0.15.10

any thoughts?

**Comment by Unitech** [comment]
Could you do a `pm2 update` and try again

**Comment by rogr** [comment]
Is already updated if i execute: `pm2 show 0` the `node v8 arguments` column is empty.

**Comment by rogr** [comment]
well i don't know what happened here, i even did a clean install.

I fixed it by stop using `node 0.12` and move to `4.0`

**Comment by kay-is** [comment]
I got this error with node v4 too.

When I switched the string to an array of strings it worked

    node_args: [
      '--harmony',
      '--use_strict'
    ]
